NeuroID/src/androidLib/java/com/neuroid/tracker/utils/NIDViewsExtensions.kt

```diff
@@ -39,9 +39,14 @@ private fun getParentsOfView(layers: Int, view: View): String {
     if (view.parent == null) {
         throw Error("Before registerTarget add the view to his parent")
     }
-    val childView = view.parent as View
-    return if (layers == 3 || childView.id == android.R.id.content) "" else {
-        "${childView.javaClass.simpleName}/${getParentsOfView(layers + 1, childView)}"
+    return if (view.parent is View) {
+        valew = view.parent as View
```

Comment:
2 q's, is it `valew` or `childView`? and do we need this? I don't see it used anywhere?

Reply:
yikes, thanks for that, incorrect syntax there. should be fixed now.